server/views/pages/manageConditions/outOfBoundsPremises/list.njk

```diff
@@ -0,0 +1,121 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set pageTitle = applicationName + " - Create a licence - Locations" %}
+{% set pageId = "licence-locations-list-page" %}
+{% set fromReviewAppend = "&fromReview=true" if fromReview %}
+{% block content %}
+    {% set locationCount = conditions.length %}
+
+    <h1 class="govuk-heading-l">You have added {% if locationCount == 1 %}one{% else %}{{ locationCount }}{% endif %} location{% if locationCount > 1 %}s{% endif %}</h1>
+
+    {% set items = [] %}
+    {% for condition in conditionsData %}
+        {% if condition.name and condition.address %}
+            {% set hiddenText = condition.name %}
+            {% set keyHtml = 'Name or type of premises:<br/>Building and street:<br/>Town:</br>'%}
+            {% set valueHtml = condition.name + "<br/>" + condition.address['addressLine1']
+            +  ((", " + condition.address['addressLine2']) if condition.address['addressLine2'] else '')
```

Comment:
wondering if this is going to allow html injection?

Reply:
yes it does, fixed now